bin/improver-wind-downscaling

```diff
@@ -105,7 +97,7 @@ def main():
                 silhouette_roughness_filepath, sigma, target_orog,
                 standard_orog, float(args.model_resolution),
                 z0_cube=veg_roughness_cube,
```

Comment:
Does the , need removing?

Reply:
It doesn't matter in Python. I've removed it because it's unnecessary.